pkg/channel/testing/channel_mock.go

```diff
@@ -82,9 +84,11 @@ func (_m *MockAppChannel) Init() {
 
```

Comment:
(Can't leave a commend on L82) Doesn't the `Init` method need a lock too?

Reply:
Agreed, incorporated.